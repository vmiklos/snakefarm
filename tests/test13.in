debug 1
out 1
time 100
load test13.map
addplayer 1
addsnake 1 1 0 1 1 1
18 0
17 0
endsnake
addplayer 2
addsnake 2 2 3 1 1 0
19 1
12 1
endsnake
comment # az elso kigyonak meg kell halnia, a masodiknak sertetlennek maradnia
step
exit