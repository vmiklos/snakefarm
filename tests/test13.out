Event MapLoad test13.map
# az elso kigyonak meg kell halnia, a masodiknak sertetlennek maradnia
Event Step 1
StepEvent Snake_Snake 2
StepEvent SnakeDie
End Step
Event Step 2
End Step
Event Exit